packages/dx-grid-core/src/utils/group-panel.js

```diff
@@ -11,7 +11,7 @@ export const getGroupCellTargetIndex = (geometries, sourceIndex, { x, y }) => {
 
   const targetIndex = targetGeometries.findIndex((geometry, index) => {
     const inVerticalBounds = isOnTheSameLine(geometry, y);
-    const inHorizontalBounds = x >= geometry.left && x <= geometry.right;
+    const inHorizontalBounds = x >= geometry.left - 5 && x <= geometry.right + 5;
```

Comment:
Maybe I must pull out this numbers (-5/+5) level higher? And make this numbers are dependent on style indent